jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/WatchServiceTest.java

```diff
@@ -89,12 +89,13 @@ void setUp(@TempDir Path tempDir) throws IOException {
         .postExec("ls -lt /deployments")
         .build())
       .build();
+    Path path = Files.createDirectory(tempDir.resolve("target"));
     watchContext = WatchContext.builder()
       .buildContext(JKubeConfiguration.builder()
         .project(JavaProject.builder()
           .baseDirectory(tempDir.toFile())
           .build())
-        .outputDirectory(Files.createDirectory(tempDir.resolve("target")).toFile().getAbsolutePath())
+        .outputDirectory(path.toFile().getName())
```

Comment:
As stated previously, for clarity, it would be better if this read as a string literal:
```suggestion
        .outputDirectory("target")
```

It's less confusing, and we understand that the value intended to go here is the name of the output directory which in this case is `target`

Reply:
Oh, I'm sorry, I thought you meant to say that I need to make it more readable by separating creation from assignation. No problem, I'll get to it.

Reply:
awesome, thx